installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -59,14 +60,16 @@ do
 
   if [[ $(inArray "${lambda_services[@]}" "$element") ]]; then
 			service_type="function"
-			if [[ $(inArray "${nodejs61_service[@]}" "$element") ]]; then			
+      deployment_targets={"function": {"S": "aws_lambda"}}
+			if [[ $(inArray "${nodejs61_service[@]}" "$element") ]]; then
 				provider_runtime="nodejs6.10"
 			else
 				provider_runtime="nodejs4.3"
 			fi
  else
 			service_type="api"
-			if [[ $(inArray "${nodejs61_service[@]}" "$element") ]]; then	  	
+      deployment_targets={"api": {"S": "aws_apigateway"}}
+			if [[ $(inArray "${nodejs61_service[@]}" "$element") ]]; then
 				provider_runtime="nodejs6.10"
 			else
```

Comment:
Not related to your changes, but can you add a TODO comment here? This whole conditional needs work, needs to be simplified/hardcoded runtime versions need to be removed where possible.

Reply:
yes will create an issue and add it. I have noticed this in multiple places(jenkins build packs for scms, runtimes, etc). There are lot of these things that needs fixing and we need to really do it otherwise it will become a nightmare managing this(like C# or Go that we want to support)